server/vars/vars.go

```diff
@@ -23,7 +23,7 @@ const (
 	UptimeVarName = "uptime"
 
 	// The name of the product
-	Product = "kapacitor"
+	Product = "kapacitor oss"
```

Comment:
This will break things. the product is used when making calls to various third parties, if we change the name we risk breaking their integrations.